chain/indexer/README.md

```diff
@@ -70,3 +67,35 @@ $ cargo run --release ~/.near/betanet
 ```
 
 After the network is synced, you should see logs of every block produced in Betanet. Get back to the code to implement any custom handling of the data flowing into the indexer.
+
+## Tweaks
+
+By default, nearcore is configured to do as little work as possible while still operating on an up-to-date state. Indexers may have different requirements, so there is no solution that would work for everyone, and thus we are going to provide you with the set of knobs you can tune for your requirements.
+
+As already has been mentioned in this README, the most common tweak you need to apply is listing all the shards you want to index data from; to do that, you should ensure that `"tracked_shards"` in the `config.json` lists all the shard IDs, e.g. for the current betanet and testnet, which have a single shard:
+
+```
+...
+"tracked_shards": [0],
+...
+```
+
+Another tweak changes the default "fast" sync process to a "full" sync process. When the node gets online and observes that its state is missing or outdated, it will do state sync, and that can be done in two strategies: (1) ("fast" / default) sync all the block headers to ensure that the chain is valid, that means that the node won't have any data, only the proofs, so Indexer will skip these blocks, (2) (very slow / full sync) sync all the blocks from the genesis. To force full sync, make the following change to your `config.json`:
```

Comment:
"won't have any data" is too strong. For example the node will have the necessary state after syncing, it's just that indexer won't because of the way it tracks data.

Reply:
Good point! I will re-write it.